cordformation/src/main/kotlin/net/corda/plugins/Dockerform.kt

```diff
@@ -201,6 +212,29 @@ open class Dockerform @Inject constructor(objects: ObjectFactory) : Baseform(obj
             services[it.containerName] = service
         }
 
+        if (external.containerImage.isPresent()) {
```

Comment:
You have given the `containerImage` property a convention of `""`. Gradle will therefore _always_ consider this property to be present. I think what you need to do here is remove the convention and declare the property as `@Optional`.